### PR TITLE
Upgrade clr 1.0.3 issue

### PR DESCRIPTION
**Pull request by jwangyangls**

### CODE REVIEW

File: src/portal/lib/src/tag/tag-detail.component.html [hunk 1]
```diff
@@ -34,7 +34,8 @@ <h2 class="custom-h2">{{repositoryId}}:{{tagDetails.name}}</h2>
                         </section>
                         <section class="detail-row">
                             <label class="detail-label">{{'TAG.SCAN_COMPLETION_TIME' | translate }}</label>
-                            <div class="image-details" [title]="scanCompletedDatetime | date">{{scanCompletedDatetime | date}}</div>
+                            <div class="image-details" [title]="scanCompletedDatetime | date">{{scanCompletedDatetime |
+                                date}}</div>
```
Comment (ninjadq) [inline src/portal/lib/src/tag/tag-detail.component.html]: why data filter start with a new line?

File: src/core/service/notifications/registry/handler.go [hunk 2]
```diff
@@ -173,12 +173,11 @@ func filterEvents(notification *models.Notification) ([]*models.Event, error) {
 }
 
 func checkEvent(event *models.Event) bool {
-	// pull and push manifest by docker-client or vic or jib
-	if (strings.HasPrefix(event.Request.UserAgent, "docker") || strings.HasPrefix(event.Request.UserAgent, vicPrefix) || strings.HasPrefix(event.Request.UserAgent, "jib")) &&
-		(event.Action == "pull" || event.Action == "push") {
+	// pull and push manifest
+	if strings.ToLower(strings.TrimSpace(event.Request.UserAgent)) != "harbor-registry-client" && (event.Action == "pull" || event.Action == "push") {
```
Comment (ninjadq) [inline src/core/service/notifications/registry/handler.go]: is this code change caused by a wrong rebase action?

File: src/portal/lib/src/tag/tag-detail.component.html (hunk 1 above)
Comment (jwangyangls) [reply src/portal/lib/src/tag/tag-detail.component.html]: resolve it

File: src/core/service/notifications/registry/handler.go (hunk 2 above)
Comment (jwangyangls) [reply src/core/service/notifications/registry/handler.go]: This is not a change I made.

**Pull request merged**